tests/pos-macros/i4774b.scala

```diff
@@ -4,7 +4,7 @@ import scala.quoted._
 object Test {
   def loop[T](x: Expr[T])(implicit t: Type[T], qctx: QuoteContext): Expr[T] = '{
     val y: t.Underlying = $x;
-    ${loop[t.Underlying]( // error
+    ${loop[t.Underlying](
```

Comment:
This was actually crashing with an assertion failure at
https://github.com/lampepfl/dotty/pull/10110/commits/8afec6683dac5a1bf3cea51d762d4b6290371fb2#diff-b3069d0753da3a2c12847b53d12eaf5d5003af1d6e6438c84f1dd5f6ef6a668eL218.

This also happens on master.

Reply:
I did not open an issue as the fix was straightforward.